packages/botonic-plugin-google-translate/README.md

```diff
@@ -0,0 +1,65 @@
+# Botonic Plugin Google Translate
+
+## What does this plugin do?
+This plugin uses [Google Cloud Translation API](https://cloud.google.com/translate) to detect the language of the user input text.
```

Comment:
specify which iso is used. Since we set in locale field, be explicit here on whether the country is ever set (only "es" or sometimes "es_ES"?

Reply:
since the plugin contains the word translate, make explicit that the plugin does not yet have an API to translate text